languages/go/polar.go

```diff
@@ -37,7 +37,7 @@ func newPolar() (*Polar, error) {
 
 	builtinClasses := map[string]reflect.Type{
 		"Boolean":    reflect.TypeOf(true),
-		"Integer":    reflect.TypeOf(int(1)),
+		"Integer":    reflect.TypeOf(int64(1)),
```

Comment:
is this change required due to the new instance checking behavior or is it unrelated?

Reply:
Required